components/Blocks/CTABlock.js

```diff
@@ -1,10 +1,10 @@
-import React, {PropTypes} from 'react';
+import React, { PropTypes } from 'react';
```

Comment:
One of the rules that we were (accidentally) breaking from airbnb's style guide is [object-curly-spacing](http://eslint.org/docs/rules/object-curly-spacing.html).